app/views/admins/_user_entry.haml

```diff
@@ -37,6 +37,11 @@
           %dd= user.username
           %dt= t('.email')
           %dd= user.email
+          %dt= t('.role.label')
+          %dd
+            = t('.role.spotlight') if Role.spotlight? user.person
+            = t('.role.moderator') if Role.moderator? user.person
+            = t('.role.admin') if Role.is_admin? user.person
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.